scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -619,3 +619,86 @@ def test_inv_single_rotation():
 
     assert_array_almost_equal(result1, eye3d)
     assert_array_almost_equal(result2, eye3d)
+
+
+def test_apply_single_rotation_single_point():
+    r_1d = Rotation.from_dcm([
+        [0, -1, 0],
+        [1, 0, 0],
+        [0, 0, 1]
+    ])
+    r_2d = Rotation.from_dcm([[
+        [0, -1, 0],
+        [1, 0, 0],
+        [0, 0, 1]
+    ]])
+
+    v_1d = np.array([1, 2, 3])
+    v_2d = np.array([[1, 2, 3]])
```

Comment:
Same here, but really minor.

Reply:
`np.atleast_2d` has the same issue as above, using `np.expand_dims`.